docs/README.md

```diff
@@ -37,44 +39,44 @@ Or to a github branch (e.g. `reference_proto`) of any public github repository (
 https://www.cyberbotics.com/doc/guide/index?version=remidhum:reference_proto
 ```
 
-## Run the doc locally
+## To Run the doc locally,follow these steps:
```

Comment:
In English a space is added after a coma sign.

Reply:
Thanks for pointing that out . I'll correct that immediately